opg-php-fpm-71-ppa-1604/docker/my_init.d/40-update-php-fpm-environment

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""
+Iterates over env variables and for each non empty variable create entry in
+    /etc/php/7.1/fpm/pool.d/app.conf
+
+"""
+import os
+
+skip_variables = ["LS_COLORS"]
+
+
+with open("/etc/php/7.1/fpm/pool.d/app.conf", "a") as app_conf:
+
+    for name, value in os.environ.iteritems():
```

Comment:
Are we sure we want all env values passed through to PHP?

Reply:
Think this has been a hangover from the original php-fpm scripts, we probably need to look at it in all php-fpm images

Reply:
I think it's best to least this for now then. Extra env vars shouldn't be a problem.

Reply:
Created a story in my current sprint to investigate it and possibly update

Reply:
ah, I've just removed it as i couldnt see any relevant variable to services we are going to switch to use this container. if i notice any issue around missing env vars i will put the script back but will make the skip-list more explicit